src/elements/button-group/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@uswitch/trustyle.button-group",
+  "version": "0.0.0",
```

Comment:
is version 0.0.0 ok to have?

Reply:
will be updated by `npx lerna version` before merging :)